core/templates/dev/head/services/explorationContextService.js

```diff
@@ -34,7 +34,7 @@ oppia.factory('explorationContextService', [
   function($window, PAGE_CONTEXT, EDITOR_TAB_CONTEXT) {
     var _pageContext = null;
     var _explorationId = null;
-
+    var isVideoPlayedList = {};
```

Comment:
Looking at this in context, I think it would actually be better to create a separate service altogether, such as seenVideosService, and move the video-related stuff there. The reason for this is that it looks like this service is more about things 'surrounding' the exploration, so putting the video list in here seems a bit odd, although it functionally works. Sorry for not realizing this earlier.

Also I would recommend renaming this to something like seenVideos or seenVideosDict -- ending with 'List' suggests that the data structure this represents is an array.